python/ola/ClientWrapperTest.py

```diff
@@ -208,7 +209,16 @@ class results:
 
     def DataCallback(self):
       data = sockets[1].recv(4096)
-      self.assertTrue(len(data) > 100)
+      expected = binascii.unhexlify(
+        "7d000010080110001a0d557064617465446d784461746122680801126400000"
+        "000000000000000000000000000000000000000000000000000000000000000"
+        "000000000000000000000000000000000000000000000000000000000000000"
+        "000000000000000000000000000000000000000000000000000000000000000"
+        "000000")
+      self.assertEqual(data, expected,
+                       msg="Regression check failed. If protocol change "
```

Comment:
Personally I don't find the standard output at all useful but in theory at least it shows where there's a difference.  But I thought printing out the "change the test to AAAA if you meant to do this" was the most helpful and leave the standard in case it helps something.

Reply:
I suspect it won't be generally, but you should be able to paste it (or part of it) into http://rdm.openlighting.org/tools/packet-extractor for example to see what's going on I think. Plus it's always nice to have a bit of a sanity check of values; it's not actually just an empty string for example or a single byte.